frontend/src/views/SubmissionsView.vue

```diff
@@ -32,7 +34,11 @@ onMounted(async () => {
       />
     </TabPanel>
     <TabPanel header="Statistics">
-      <SubmissionStatistics />
+      <SubmissionStatistics
+        v-if="statistics"
```

Comment:
Could lift the v-if conditional up to the TabPanel component to not render to DOM if AC permits it - there could be rare cases where the tab is still there, but there's no content in there since statistics is undefined. May need to consider a v-else and a quick "No statistics available" div alternative to cover this unlikely state.